weave/legacy/ops_domain/artifact_tag_ops.py

```diff
@@ -0,0 +1,29 @@
+import typing
+
+from weave.api import op
+
+from weave.legacy.ops_domain import wb_domain_types as wdt
+
+
+# Section 1/6: Tag Getters
+# None
+
+# Section 2/6: Root Ops
+# None
+
+# Section 3/6: Attribute Getters
+
+@op(
+    name="artifactTag-name",
+)
+def op_artifact_tag_name(tag: wdt.ArtifactTag) -> str:
+    return tag.name
```

Comment:
note: similar to the existing `artifactAlias-alias` op from: https://github.com/wandb/weave/blob/69710f9342db3b0fbb3f46ad64b3d380296ef71c/weave/legacy/ops_domain/artifact_alias_ops.py#L22